src/disco/components/Addon/index.js

```diff
@@ -327,16 +335,13 @@ export function mapStateToProps(state, ownProps) {
   // `ownProps.guid` is already "normalized" with `getGuid()` in the
   // `DiscoPane` container component.
   const installation = state.installations[ownProps.guid] || {};
-  const addon = getAddonByGUID(state, ownProps.guid);
```

Comment:
This is not needed because `DiscoPane` passes the `addon` object to each `Addon` component.